sklego/meta/outlier_classifier.py

```diff
@@ -52,6 +52,33 @@ def fit(self, X, y=None):
         ValueError
             - If the underlying model is not an outlier detection model.
             - If the underlying model does not have a `decision_function` method.
+
+        Example
```

Comment:
Hey @anopsy, I have the same feedback as for `DictMapper`: if you could move the example up in the docstring I think it would be easier and faster for folks to find when scrolling through the api documentation without the need to step down into the `.fit(..)` method.

I think this example is ready to merge after that change

Reply:
I will do that!